capactio/capact
# Move manifest generation out of `alpha` commands

## Description

Changes proposed in this pull request:

- Move manifest generation out of `alpha` commands

> **NOTE:** Documentation check is failing because of `Error: reviewdog: Too many results (annotations) in diff.`

## Testing

1. Check out the PR
1. Build the binary and run sample commands:

```
# for macOS
make build-tool-cli && mv ./bin/capact-darwin-amd64 /usr/local/bin/capact

capact manifest generate
capact manifest generate interface cap.interface.foo.bar
```

## Reason

As there were significant changes in the manifestgen tool, with feature coverage for all manifest kinds, there's no need to still keep it under `alpha` commands.
